src/pages/home/report/ReportActionsView.js

```diff
@@ -239,12 +285,17 @@ class ReportActionsView extends React.Component {
         needsLayoutCalculation,
     }) {
         return (
-            <ReportActionItem
-                action={item.action}
-                displayAsGroup={this.isConsecutiveActionMadeByPreviousActor(index)}
-                onLayout={onLayout}
-                needsLayoutCalculation={needsLayoutCalculation}
-            />
+            <View>
```

Comment:
Maybe just use a fragment `<>` here instead of a `<View>`?

Reply:
Looks like my presumption was correct (that someone would ask this). I wrote about it on my third note in the PR description.

Native and web/desktop have different `<InvertedFlatList />`, and this workaround caused them to have this weird behavior where, if we insert just a fragment, the indicator will top on the top of the message (correct place) on web/desktop, but actually on the bottom of the element on native. Has to do with the `scaleY(-1)` and all the inverted DOM stuff from the FlatList.
That's the reason I wrapped in a `<View />` instead.

Reply:
Oh, sorry I missed that in the description.

Reply:
Could you please add a comment with this explanation of why we are using a `<View>` instead of a fragment so that future engineers don't assume they can just change it.

Reply:
@bondydaa Good point. Added a comment linking to this review.